lib/lapis_lazuli/browser.rb

```diff
@@ -365,12 +341,13 @@ def create_driver(browser_wanted=nil, optional_data=nil)
             else
               world.log.debug "User-agent was already set in the :switches."
             end
-          else
-            warn "#{device} user agent cannot be set for #{b.to_s}. Only Chrome & Firefox are supported."
-        end
+         # else
```

Comment:
lets remove all the commented out code, or do we still need this at some point?

Reply:
Fixed.